src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -1096,35 +1090,35 @@ namespace InternalHeatGains {
                 }
             }
 
-            for (int peopleNum = 1; peopleNum <= state.dataHeatBal->TotPeople; ++peopleNum) {
+            for (int peopleNum2 = 1; peopleNum2 <= state.dataHeatBal->TotPeople; ++peopleNum2) {
```

Comment:
Lots of these:

[src/EnergyPlus/InternalHeatGains.cc:1099]:(style),[shadowVariable],Local variable 'peopleNum' shadows outer variable